### DISCUSSION TITLE
[Feature Request]: Error Pages

### DISCUSSION THREAD

**Discussion by danielschuster-muc** [danielschuster-muc/potter-db#172] [Ideas]
### Project

Website (Potter DB Search)

### Describe the feature you want to request

Create Custom Error Pages (404/500) in a Harry Potter Style. If you don't know how to implement those pages, you can also show a screenshot of a possible layout for those pages.

**Comment by naXa777** [comment]
Idea 1.

Copy https://www.wizardingworld.com/404 for 404 page

Idea 2.

It feels like this art will also fit well for 404 pages

![image](https://user-images.githubusercontent.com/3858548/193477415-cabc7239-09c8-48dc-aa06-3f8a04431101.png)

**Reply by danielschuster-muc** [reply]
The second idea sounds really cool! 
If anyone want to make a 404 page of the 2. idea, feel free to!

**Comment by danielschuster-muc** [comment]
For the 500 page I was thinking of something like this: 
![image](https://user-images.githubusercontent.com/90754725/194037336-0c29511a-21c1-48dd-8718-3bb02430125c.png)

**Comment by danielschuster-muc** [comment]
I have created 2 issues for this if someone wants to work on the error pages: #202 and #203
